client/src/components/MyPage/ClubListSetting.tsx

```diff
@@ -1,28 +1,120 @@
+import styled from 'styled-components';
+import { S_TagSmall } from '../UI/S_Tag';
+import { S_Description, S_Label, S_SmallDescription } from '../UI/S_Text';
+import { ClubData } from '../../types';
+import getGlobalState from '../../util/authorization/getGlobalState';
+import leaderBadgeIcon from '../../assets/icon_leader-badge.svg';
+import { S_NegativeButton, S_SelectButton } from '../UI/S_Button';
+import { useEffect, useState } from 'react';
+import { getFetch, deleteFetch } from '../../util/api';
 
-function ClubListSetting() {
-  const data = clubDataAll.data;
-  console.log(data); // API로 가져올 데이터 임시
+const S_ClubBox = styled.div`
+  // 전체 컨테이너
+  display: flex;
+  padding: 20px 0px;
+  background-color: var(--white);
+  border-top: 1px solid var(--gray100);
+`;
+
+const S_TitleBox = styled.div`
+  // 제목 - 소모임 설정 버튼 정렬
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  justify-content: space-between;
+`;
+const S_ImgBox = styled.div<{ img?: string }>`
+  // 클럽 대표 이미지 썸네일크기로 자르기
+  margin-right: 15px;
+  min-width: 80px;
+  height: 80px;
+  border-radius: 10px;
+  background-size: cover;
+  background-position: center center;
+  background-image: url(${(props) => props.img});
+`;
+const S_ContentsBox = styled.div`
+  // 제목 / 카테고리, 지역, 인원 / 설명 / 태그 박스
+  width: 100%;
+  .tagbox {
+    margin-bottom: 20px;
+  }
+  .settingbox {
+    padding: 10px 0px;
+    border-top: 1px solid var(--gray100);
+  }
+`;
+const S_Hidden = styled.div`
+  // 설명글 길어지면 잘라주기
+  display: -webkit-box;
+  word-wrap: break-word;
+  -webkit-line-clamp: 1;
+  -webkit-box-orient: vertical;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  margin-bottom: 5px;
+`;
+
+interface ClubListSettingProps {
+  userClubId?: number;
+  clubRole?: string | null;
+}
+
+function ClubListSetting({ userClubId, clubRole }: ClubListSettingProps) {
+  const navigate = useNavigate();
+  // 받아온 userClubResponses.userClubId로 get 요청 보내기
+  // 요청보낼 URI는 API 문서 28번 '소모임 단건 조회'
+  const [club, setClub] = useState<ClubData>();
+
+  useEffect(() => {
+    getFetch(`${process.env.REACT_APP_URL}/clubs/${userClubId}`).then((data) => {
+      setClub(data.data);
+    });
+  }, []);
 
   return (
-    <div>
-      {data.map((el) => (
-        <ClubList
-          key={el.clubId}
-          clubId={el.clubId}
-          clubName={el.clubName}
-          clubImage={el.profileImage}
-          content={el.content}
-          local={el.local}
-          categoryName={el.categoryName}
-          memberCount={el.memberCount}
-          tagResponseDtos={el.tagResponseDtos}
-        />
-      ))}
-    </div>
+    <S_ClubBox>
+      <S_ImgBox img={club?.clubImage} />
+      <S_ContentsBox>
+        <Link to={`/club/${club?.clubId}`}>
+          <S_TitleBox>
+            <S_Label>
+              {club?.clubName}
+              {/* 롤이 리더인 경우 아이콘 보여주기 */}
+              {clubRole === 'LEADER' ? <img src={leaderBadgeIcon} alt='소모임장 아이콘' /> : null}
```

Comment:
```suggestion
              {clubRole === 'LEADER' && <img src={leaderBadgeIcon} alt='소모임장 아이콘' />}
```
이렇게 바꾸면 굳이 : null 을 안 써줘도 돼요

Reply:
오 관습적으로 삼항연산자 쓰던게 남아있었군요... 감사합니다